docs/content/documentation/getting-started/installation.md

```diff
@@ -49,12 +49,20 @@ $ sudo dpkg -i zola_<version>_amd64_debian_<debian_version>.deb
 
 ### Fedora
 
-Zola has been available in the official repositories since Fedora 29.
+Zola has been available in the official repositories since Fedora 29. (unmaintained)
 
 ```sh
 $ sudo dnf install zola
```

Comment:
Can we remove that? Looking at the discussion on the fedora project it's not going to be maintained anytime soon.

Reply:
I this yes